MainWindow.xaml.cs

```diff
@@ -775,9 +881,33 @@ private void ChapterPropsButton_Click( object sender, RoutedEventArgs e )
 			cw.ShowDialog();
 		}
 
-		private void TileEditButton_Click( object sender, RoutedEventArgs e )
-		{
-			Chapter c = ((Button)e.Source).DataContext as Chapter;
+        private void TileEditButton_Click(object sender, RoutedEventArgs e)
+        {
+            OpenChapterEditor(((Button)e.Source).DataContext as Chapter);
+        }
+
+        private void OpenTokenEditor(Token t)
```

Comment:
At some point these should get wired up to work for SquareTile too.

Reply:
True, I can test this with some of the test scenarios so I'll fix it right away